packages/govern-server/src/types.ts

```diff
@@ -0,0 +1 @@
+export type Address = string
```

Comment:
Not sure if this type is required if we do not any further parsing of the passed URL 🤷‍♂️

Reply:
It’s just for reading / documentation purposes, since we use those so much :slightly_smiling_face:

Reply:
Yup, true that.. this will definitely increase the readability.